index.js

```diff
@@ -133,11 +133,13 @@ var createAssertion = function createAssertion(type, message, wrappers, block, m
 	if (descriptors.length === 0) {
 		throw new RangeError(inspect(type) + ' called with no wrappers defined');
 	}
-	var describeMsgs = [];
-	forEach(wrappers, function (wrapper) {
-		checkThis(wrapper);
-		describeMsgs.push(getThisDescription(wrapper));
```

Comment:
I changed extend to apply the description to the descriptor object in all cases, so we don't need to look at wrappers anymore, just at the flattened `descriptors`.